retro/retro_env.py

```diff
@@ -191,8 +208,8 @@ def step(self, a):
         if self.movie:
             self.movie.step()
         self.em.step()
-        self.img = ob = self.get_screen()
```

Comment:
`self.img` still needs to be updated for `env.render()` to work, further the check at the beginning of `env.step` will need to be updated.

Reply:
Good call, thanks for the review!